CHANGELOG.md

```diff
@@ -7,11 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Change
+
+* **Dynamic Schema** Refact the carousel to have a dynamic schema.
```

Comment:
⛏ "Refactor"

Reply:
Refactoring*?

Reply:
It's done, so I think that is "Refactored" 

https://www.quora.com/Is-it-grammatically-correct-to-say-I-refactored-my-code